src/theme/MuiThemeProvider/muiTheme.ts

```diff
@@ -121,6 +121,7 @@ export default createTheme({
     h4: {
       fontSize: '1.25rem',
       fontWeight: 600,
+      lineHeight: '1.875rem',
```

Comment:
This should be the default line-height of all text elements (1.5)

Reply:
It's not larger: 30 is 20 * 1.5, 1.5 being the line height we apply to all elements